### PR TITLE
[new] Add rxdb w/ npm auto-update via single package.json

### LINKED ISSUE #12964: [Request] Add rxdb
**Library name:** rxdb
**Git repository url:** https://github.com/pubkey/rxdb
**npm package name or url** (if there is one): https://www.npmjs.com/package/rxdb
**License (List them all if it's multiple):** [Apache-2.0](https://github.com/pubkey/rxdb/blob/master/LICENSE.txt)
**Homepage**: https://pubkey.github.io/rxdb/

It's recommended to use npm auto-update to fetch files for a browser.

### PR DESCRIPTION
**Pull request by alemuro**
close #12964, cc @pubkey

# Pull request template for new library

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/pubkey/rxdb
* NPM package url (optional): https://www.npmjs.com/package/rxdb
* License and its reference: [Apache-2.0|https://github.com/pubkey/rxdb/blob/master/LICENSE.txt]
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 189
  * Count of stars: 8231
  * Count of forks: 3732
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week: 1318
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [X] The lib has notable popularity
  * [X] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [X] More than 800 downloads stats per month on npm registry
* [X] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [ ] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [x] Close corresponding issue in commit message
* [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by alemuro** [comment]
Hello! 😄 
This is my first contribution to an Open Source project! I hope its ok, otherwise tell me something and I'll fix it ^^ 
Have a nice day

File: ajax/libs/rxdb/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "rxdb",
+  "npmName": "rxdb",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.js"
```
Comment (extend1994) [inline ajax/libs/rxdb/package.json]: Let's use `!(es|lib){,**/*}` which means we want to fetch all files under `basePath` other than the files under `es`/`lib`.

File: ajax/libs/rxdb/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      ]
+    }
+  ],
+  "filename": "rxdb.min.js",
```
Comment (extend1994) [inline ajax/libs/rxdb/package.json]: `rxdb.browserify.min.js` is the correct one.

**Comment by extend1994** [comment]
@alemuro No problem :)

**Comment by PeterDaveHello** [comment]
Maybe we don't need rxdb.webpack.js & rxdb.js in the old versions? I'll manually exclude them first, point me if I'm wrong and feel free to make it right.

**Pull request merged**